test/unit/app/tools/test_evaluation.py

```diff
@@ -45,7 +47,7 @@ def setUp(self):
         self.job.history = History()
         self.job.history.id = 42
         self.job.parameters = [JobParameter(name="thresh", value="4")]
-        self.evaluator = ToolEvaluator(self.app, self.tool, self.job, self.test_directory)
+        self.evaluator = ToolEvaluator(cast(MinimalToolApp, self.app), self.tool, self.job, self.test_directory)
```

Comment:
Maybe do the cast in `lib/galaxy/app_unittest_utils/tools_support.py::UsesApp.setup_app()` instead?